### PR TITLE
Talk Product Guide - features and configuration

### PR DESCRIPTION
**Pull request by kgardnr**
## Contents

- [x] Reader and Commenter Features
- [x] Moderator Features
- [x] Configuring Talk

### CODE REVIEW

**Comment by kgardnr** [comment]
@wyattjoh ready for your review!

File: docs/_docs/03-02-product-guide-commenter-features.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+permalink: /commenter-features/
+---
+
+## Signing up for Talk
+
+There are 2 ways that newsrooms can support signup/login functionality with Talk:
+
+Use Talk’s auth plugin out of the box (supports account registration with username and password, as well as features like forgot password)
+
+Create their own auth plugin (you can use ours as a template) to integrate with your own auth systems
```
Comment (wyattjoh) [inline docs/_docs/03-02-product-guide-commenter-features.md]: Our auth plugin won’t work to store state unless they open a pop up. I’ll update this.

File: docs/_docs/03-02-product-guide-commenter-features.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+### Permalinks
+
+All levels of comments and replies are able to be linked to via permalink. Permalinks are structured using a `commentId` query param:
+
+https://domain.com/article-asset-slug/?commentId=0879edd7-1c4f-4918-abd2-a770db446b75
+
+### Threading
+
+Talk supports by default 3 levels of threading, meaning each top-level comment has a depth of 3 replies; replies beyond that are not nested below the 3rd level. You can adjust this using the `TALK_THREADING_LEVEL` configuration variable. We don’t recommend deep threading because it can cause issues with styling, especially on mobile.
```
Comment (wyattjoh) [inline docs/_docs/03-02-product-guide-commenter-features.md]: Link whenever referenced to the actual config param docs.

File: docs/_docs/03-02-product-guide-commenter-features.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+## Badges
+
+Badges differentiate users and comments on the stream. By default, Talk has two badges. 
+
+The Staff user badge that shows when a commenter has an Admin, Moderator, or Staff role. 
+
+The Featured comment badge shows when a comment has been featured.
+
+Another optional badge is the Subscriber badge (which is available as a [Recipe](/plugin-recipes/#recipe-subscriber).
```
Comment (wyattjoh) [inline docs/_docs/03-02-product-guide-commenter-features.md]: Links should include the relative URL liquid template tag

File: docs/_docs/03-03-product-guide-moderator-features.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+This shows the number of comments that a commenter has made that currently display on the site.
+
+**Reject Rate**
+
+This shows the % of comments a commenter has had rejected by moderators, or automatically.
+
+**Reports**
+
+This shows if a commenter is a reliable flagger, an unreliable flagger, or a neutral flagger. [Read more about reliable and unreliable flaggers here](LINK).
```
Comment (wyattjoh) [inline docs/_docs/03-03-product-guide-moderator-features.md]: Missing link?

File: docs/_docs/03-03-product-guide-moderator-features.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+#### Moderating Usernames
+
+Any usernames that have been reported will show in the Reported Usernames sub-tab. Moderators can approve usernames if they’re suitable, or reject a username. If a username is rejected, the commenter will be notified that they need to change their username; until they do, they will be suspended from Talk. The updated username then again appears in this queue for a decision by moderators.
+
+#### Managing People & Roles
+
+All your team and commenters show in the People sub-tab. From here, you can manage your team members’ roles (Admins, Moderators, Staff), as well as search for commenters and take action on them (e.g. Ban/Un-ban, Suspend, etc.). 
+### Configure
+
+See [Configuring Talk](LINK).
```
Comment (wyattjoh) [inline docs/_docs/03-03-product-guide-moderator-features.md]: Missing link?

File: docs/_docs/03-06-product-guide-configuring-talk.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+Allows toggling pre-moderation for the current comment stream.
+
+### Pre-moderation Comments Containing Links
+
+Allows toggling of pre-moderating comments that have links.
+
+### Ask Readers a Question & Question Icons
+
+Admins can choose to Ask Readers a Question in order to help guide the discussion. [Read more about why this is important on our blog](https://blog.coralproject.net/the-empty-box/).
```
Comment (wyattjoh) [inline docs/_docs/03-06-product-guide-configuring-talk.md]: External links need target=_blank

File: docs/_docs/06-01-developer-tutorials.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
+---
+title: Developer Tutorials
+permalink: /developer-tutorials/
+---
+
+Coming soon!
```
Comment (wyattjoh) [inline docs/_docs/06-01-developer-tutorials.md]: Maybe skip out inclusng this page?

**Review by wyattjoh — CHANGES_REQUESTED** [review]

File: docs/_docs/03-02-product-guide-commenter-features.md (hunk 1 above)
Comment (kgardnr) [reply docs/_docs/03-02-product-guide-commenter-features.md]: Ok!

File: docs/_docs/03-02-product-guide-commenter-features.md (hunk 3 above)
Comment (kgardnr) [reply docs/_docs/03-02-product-guide-commenter-features.md]: Done!

File: docs/_docs/03-03-product-guide-moderator-features.md (hunk 4 above)
Comment (kgardnr) [reply docs/_docs/03-03-product-guide-moderator-features.md]: Done!

File: docs/_docs/03-03-product-guide-moderator-features.md (hunk 5 above)
Comment (kgardnr) [reply docs/_docs/03-03-product-guide-moderator-features.md]: Done!

File: docs/_docs/03-06-product-guide-configuring-talk.md (hunk 6 above)
Comment (kgardnr) [reply docs/_docs/03-06-product-guide-configuring-talk.md]: Done!

File: docs/_docs/06-01-developer-tutorials.md (hunk 7 above)
Comment (kgardnr) [reply docs/_docs/06-01-developer-tutorials.md]: Done!

**Comment by kgardnr** [comment]
@wyattjoh ready for another review!

File: docs/_docs/03-02-product-guide-commenter-features.md (hunk 1 above)
Comment (wyattjoh) [reply docs/_docs/03-02-product-guide-commenter-features.md]: Just correct the second paragraph here to read:

> Create their own auth plugin to integrate with your own auth systems

This is pending a link to the "creating a custom auth plugin" section.

Comment (kgardnr) [reply docs/_docs/03-02-product-guide-commenter-features.md]: Fix forthcoming!

**Review by wyattjoh — APPROVED** [review]

**Pull request merged**